src/feedback_forensics/app/utils.py

```diff
@@ -52,180 +50,6 @@ def load_json_file(file_path: str | Path) -> dict:
     return content
 
 
-def get_value_from_json(file_path: str | Path, key_path: str) -> any:
-    """Get a value from a JSON file by following a dot-notation key path without loading the entire file.
-
-    This function uses ijson to parse the JSON file incrementally, which is memory efficient
-    for large JSON files. It follows the dot-notation key path to find the desired value.
```

Comment:
nice to get rid of additional complexity!

Reply:
Yes! Although the PR adds some as well, it's nice to balance it a bit